src/platform/tls_mitls.c

```diff
@@ -1548,6 +1548,20 @@ CxPlatTlsOnCertVerify(
         goto Error;
     }
 
+    if (TlsContext->SecConfig->Flags & QUIC_CREDENTIAL_FLAG_CUSTOM_CERTIFICATE_VALIDATION) {
+        if (!TlsContext->SecConfig->Callbacks.CertificateReceived(
+                TlsContext->Connection)) {
+            QuicTraceEvent(
+                TlsError,
+                "[ tls][%p] ERROR, %s.",
+                TlsContext->Connection,
+                "Custom certificate validation failed");
+        } else {
+            Result = TRUE;
+        }
+        goto Error;
```

Comment:
Since this is now hit in both error and non error cases, we likely should rename this to `Exit`

Reply:
I think it's fine. Most of the places go there on error, so I'd just prefer to keep it as is.